Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -746,14 +746,32 @@ bool function runWave( int waveIndex, bool shouldDoBuyTime )
 	}
 
 	//SetGlobalNetTime("FD_nextWaveStartTime",Time()+10)
-	wait 10
+	if (waveIndex==0)
+	{
+		wait 14
+		//I tried to make a check of riff_minimap_state for Insane difficulty, but it didnt work so Droz & Davis will say about the minimap regardless Insane diff for now
+		PlayFactionDialogueToTeam( "fd_minimapTip" , TEAM_MILITIA )
```

Comment:
`if( FD_GetDifficultyLevel() == eFDDifficultyLevel.INSANE )`

Reply:
This way it should work, however riff_minimap_state is the playlist var that controls the minimap display on Insane difficulty and can be overriden for other difficulties. If server hosters disables it for the other difficulties, they will still be announcing the minimap ping, instead of skipping that part.

Reply:
great